docs/tutorials/intro_tutorial.rst

```diff
@@ -169,148 +206,230 @@ The beginning of both classes looks like this:
+using the ``add`` method; when we call the schedule’s ``step`` method,
+the model shuffles the order of the agents, then activates and executes
+each agent’s ``step`` method. The scheduler is then added to the model.
+
+Technical Details: The technical details about the timer object can be
+found in the `mesa
+repo <https://github.com/projectmesa/mesa/blob/main/mesa/time.py>`__.
+Mesa offers a few different built-in scheduler classes, with a common
 interface. That makes it easy to change the activation regime a given
-model uses, and see whether it changes the model behavior. This may not
-seem important, but scheduling patterns can have an impact on your
-results [Comer2014].
-
-For now, let’s use one of the simplest ones: ``RandomActivation``\ \*,
-which activates all the agents once per step, in random order. Every
-agent is expected to have a ``step`` method. The step method is the
-action the agent takes when it is activated by the model schedule. We
-add an agent to the schedule using the ``add`` method; when we call the
-schedule’s ``step`` method, the model shuffles the order of the agents,
-then activates and executes each agent’s ``step`` method.
-
-\*Unlike ``mesa.model`` or ``mesa.agent``, ``mesa.time`` has multiple
-classes (e.g. ``RandomActivation``, ``StagedActivation`` etc). To ensure
-context, time is used in the import as evidenced below with
-``mesa.time.Randomactivation``. You can see the different time classes
-as
-`mesa.time <https://github.com/projectmesa/mesa/blob/main/mesa/time.py>`__.
-
-With that in mind, the model code with the scheduler added looks like
-this:
+model uses, and see whether it changes the model behavior. The details
+pertaining to the scheduler interface can be located the same `mesa
+repo <https://github.com/projectmesa/mesa/blob/main/mesa/time.py>`__.
 
-.. code:: ipython3
-
-    import mesa
+With that in mind, the MoneyAgent code is modified below to visually
+show when a new agent is created. The MoneyModel code is modified by
+adding the RandomActivation method to the model. with the scheduler
+added looks like this:
 
+.. code:: ipython3
 
     class MoneyAgent(mesa.Agent):
         """An agent with fixed initial wealth."""
 
         def __init__(self, unique_id, model):
+            # Pass the parameters to the parent class.
             super().__init__(unique_id, model)
+
+            # Create the agent's variable and set the initial values.
             self.wealth = 1
 
         def step(self):
             # The agent's step will go here.
             # For demonstration purposes we will print the agent's unique_id
-            print("Hi, I am agent " + str(self.unique_id) + ".")
+            print(f"Hi, I am an agent, you can call me {str(self.unique_id)}.")
 
 
     class MoneyModel(mesa.Model):
         """A model with some number of agents."""
 
         def __init__(self, N):
             self.num_agents = N
+            # Scheduler is created and added to the model
```

Comment:
I'd say "assigned" instead of "added". Though I find the imperative statement like in L279 is clearer.